cmd/config/cache/help.go

```diff
@@ -28,9 +28,9 @@ var (
 		},
 		config.HelpKV{
 			Key:         Expiry,
-			Description: `cache expiry duration in days e.g. "90"`,
+			Description: `cache expiry duration in days (d), hours(h) e.g. "10d"`,
 			Optional:    true,
-			Type:        "number",
+			Type:        "string",
```

Comment:
This is a breaking change @poornas is the old format still supported?

Reply:
yes - it still defaults to days if expiry specified as a number